src/device_info.sh

```diff
@@ -0,0 +1,173 @@
+# This file deals with functions related to hardware information
+
+include "$KW_LIB_DIR/remote.sh"
+include "$KW_LIB_DIR/kwlib.sh"
+
+declare -gA shared_data=(['ram']='' ['cpu_model']='' ['cpu_frequency']='' ['disks']='' ['os']='')
+
+# This function populates the ram element from the shared_data global variable
+# with the total RAM memory from the target machine in kB
+#
+# @target Target machine
+function get_ram()
+{
+  local target="$1"
+  local ram
+  local cmd
+
+  case "$target" in
+    1) # VM_TARGET
+      echo TODO
+      ;;
+    2) # LOCAL_TARGET
+      cmd="cat /proc/meminfo | grep 'MemTotal' | grep -o '[0-9]*'"
+      ram=$(cmd_manager "SILENT" "$cmd")
```

Comment:
How about converting the memory size to G? Maybe you can have both information, kB and G.

Reply:
Maybe [`numfmt`](https://www.gnu.org/software/coreutils/manual/html_node/numfmt-invocation.html) can help here!

Reply:
uouuu... numfmt looks really nice!